Detectors/ITSMFT/common/workflow/src/STFDecoderSpec.cxx

```diff
@@ -169,8 +170,9 @@ void STFDecoder<Mapping>::run(ProcessingContext& pc)
         continue;
```

Comment:
note that sometimes this error is triggered (might be a real problem in the event building or a bit flip in the int.record), in which case you will miss a count due to the `continue`. Perhaps you should not raise an error about the wrong N_ROF in this case. E.g. you could put `nTriggersProcessed = -0x7fffffff` before `continue;`  and rain an error at L191 only if `nTriggersProcessed>0`). Also, this will protect against a false alarm at the ROF ramp up stage, where the decoding is skipped.
There is one more potential caveat: the `decodeNextTrigger` goes over the empty ROFs, so if the noise is particularly low, you may get the wrong count. 
To avoid this improbable scenario, you can move the internal counter https://github.com/AliceO2Group/AliceO2/blob/cf19c94b27953fb03d8a77132959c4ac9cd6d5e9/Detectors/ITSMFT/common/reconstruction/src/RawPixelDecoder.cxx#L117 to here: https://github.com/AliceO2Group/AliceO2/blob/cf19c94b27953fb03d8a77132959c4ac9cd6d5e9/Detectors/ITSMFT/common/reconstruction/src/RawPixelDecoder.cxx#L97 and instead of incrementing `nTriggersProcessed` yourself at L185 you just to at L162
```
int nTriggersProcessed = mDecoder->getNROFsProcessed(); // you will need to add a getter to RawPixelDecoder.h
```
and after the line 189:
```
nTriggersProcessed = mDecoder->getNROFsProcessed() - nTriggersProcessed;
```
In this case, you should assign `nTriggersProcessed = 0x7fffffff`  before line 170, to discard TF with an error.

Conditionally approving to start the CI

Reply:
Thank you for review @shahor02 , 

I implemented your comments in the new commit. I can't reproduce problematic behavior of decoder, but for normal case, locally all works fine

Reply:
Re-approving to start the CI